modules/load-balancer/variables.tf

```diff
@@ -50,7 +55,8 @@ variable "create_dns" {
 }
 
 variable "zone_name" {
-  type = string
+  type    = string
+  default = "example.com"
```

Comment:
don't add a default, add a description instead if the goal is to show format

Reply:
default is needed because it could potentially not be provided if the dns name creation is set to false

Reply:
id recommend setting default to null then

Reply:
works for me